core/io/file_access.cpp

```diff
@@ -38,7 +38,7 @@
 #include "core/io/marshalls.h"
 #include "core/os/os.h"
 
-FileAccess::CreateFunc FileAccess::create_func[ACCESS_MAX] = { nullptr, nullptr };
+FileAccess::CreateFunc FileAccess::create_func[ACCESS_MAX] = { nullptr, nullptr, nullptr };
```

Comment:
Could be initialized like this to be future proof.
```suggestion
FileAccess::CreateFunc FileAccess::create_func[ACCESS_MAX] = {};
```

Reply:
Amended the commit with this.